teamcity-settings.json

```diff
@@ -12,5 +12,6 @@
     "add_test_user": false,
     "db_annex_type": "fake",
     "instance_name": "teamcity",
-    "notify_channel": ""
+    "notify_channel": "",
+    "template_reports":[]
```

Comment:
should this not be `"template_reports": ""` (empty string rather than an empty array?)

Reply:
No because these are the parsed settings - if the user passes in a empty string on the command line, it will get saved here as an empty array

Reply:
oh of course!